src/main/views/care-standards-list.njk

```diff
@@ -31,7 +31,7 @@
 
     {{ searchInput(text = searchCases) }}
     <div class="parent-box overflow-table govuk-!-margin-top-5 search-area">
-      <table class="govuk-table" data-module="moj-sortable-table">
+      <table class="govuk-table" data-module="moj-sortable-table overflow-table">
```

Comment:
I've found a bug which is causing the sorting on the table to no longer work. I believe this is due to the overflow-table property being against the data-module - I believe this needs to be under class.